staging/src/kubevirt.io/api/core/v1/BUILD.bazel

```diff
@@ -18,6 +18,7 @@ go_library(
     importpath = "kubevirt.io/api/core/v1",
     visibility = ["//visibility:public"],
     deps = [
+        "//pkg/pointer:go_default_library",
```

Comment:
I don't think we can import `//pkg/..` packages in `/staging`.
It won't work.
Does `make && make generate` work in your local env?

Reply:
no the error that iam getting is 

```
ERROR: /root/go/src/kubevirt.io/kubevirt/staging/src/kubevirt.io/api/core/v1/BUILD.bazel:3:11: GoCompilePkg staging/src/kubevirt.io/api/core/v1/go_default_library.a failed: (Exit 1): builder failed: error executing command bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64 -tags selinux,selinux -src ... (remaining 93 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
compilepkg: missing strict dependencies:
	/root/.cache/bazel/_bazel_root/6f347497f91c9a385dcd9294645b76e0/sandbox/processwrapper-sandbox/1/execroot/kubevirt/staging/src/kubevirt.io/api/core/v1/register.go: import of "kubevirt.io/kubevirt/pkg/pointer"
No dependencies were provided.
Check that imports in Go sources match importpath attributes in deps.
ERROR: /root/go/src/kubevirt.io/kubevirt/tools/csv-generator/BUILD.bazel:3:11 GoCompilePkg tools/csv-generator/go_default_library.a failed: (Exit 1): builder failed: error executing command bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64 -tags selinux,selinux -src ... (remaining 93 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
INFO: Elapsed time: 8.968s, Critical Path: 0.35s
INFO: 2 processes: 2 internal.
FAILED: Build did NOT complete successfully
```